src/.env.test

```diff
@@ -1 +1,3 @@
+PGSQL_DATABASE_NAME='skyra-nodejs-test'
```

Comment:
And what happens, like in 99% of the cases when running tests locally, where this DB doesn't exist and the tests are run? If tests break in such a case like I suspect then find a better solution for this.

In fact, "Node -> Grpc" is an integration test, but you're hijacking the unit tests package with the tests now. "Node -> Grpc" tests should be its own complete package with its own `jest.config`, and its own `package.json`, the whole shebang.